scenarios/generate-synthetic-data/simulate-adversarial-interactions/rag/simulate_and_evaluate_rag.ipynb

```diff
@@ -0,0 +1,318 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Adversarial Simulator for a custom rag application\n",
+    "\n",
+    "## Objective\n",
+    "\n",
+    "This tutorial provides a step-by-step guide on how to leverage adversarial simulator to simulate an adversarial question answering scenario against a custom rag application.\n",
+    "\n",
+    "This tutorial uses the following Azure AI services:\n",
+    "\n",
+    "- promptflow-evals\n",
+    "- promptflow-rag\n",
```

Comment:
Consider linking to the relevant documentation

Reply:
RAG sample has been removed and will be re-added in a new PR as the `promptflow-rag` team is still working on making the cloud index work.